api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java

```diff
@@ -308,9 +311,13 @@ protected AdditionsAndRetractions parseN3ToChange(
             String lingCxt=null;
             //UQAM Taking into account the linguistic context in retract
             try {
-                lingCxt = vreq.getLocale().getLanguage();
-                if (!vreq.getLocale().getCountry().isEmpty()) {
-                    lingCxt += "-" + vreq.getLocale().getCountry();
+                if (editConfig != null && editConfig.getLiteralsInScope().get("label").get(0).getLanguage() != "") {
```

Comment:
Instead of `!=`, should likely use `!...equals()`.
..or `.isEmpty()`.

Is there a chance that the language could be null? If so, maybe `StringUtils.isBlank()`.

Reply:
good point, i changed it. I don't know if it could be null, so better catch it.